lib/Importer/ONNX.cpp

```diff
@@ -50,6 +50,26 @@ static ArgumentDictionaryTy loadArgumentMap(const onnx::NodeProto &op) {
   return dict;
 }
 
+int ONNXModelLoader::getBroadcast(const ArgumentDictionaryTy &dict) {
+  if (opsetVersion_ > 6)
+    return 1;
+  return dict.count("broadcast") ? loadInt(dict.at("broadcast")) : 0;
+}
+
+void ONNXModelLoader::setVersion(onnx::ModelProto MP) {
+  irVersion_ = MP.ir_version();
+  opsetVersion_ = 0;
+  if (irVersion_ >= 3) {
+    // Model with IR version < 3 doesn't have opset_import specified.
+    for (const auto &imp : MP.opset_import()) {
+      if (!imp.has_domain() || imp.domain() == "") {
+        opsetVersion_ = imp.version();
+        break;
+      }
+    }
```

Comment:
Is it an error if opsetVersion_ was not set after the loop?

Reply:
So according to ONNX team, there should be an assertion for irVersion_ < 3.  If there is no domain or domain is "", it means it is ONNX default op. So as long as irVersion_ >=3, the opsetVerson_ should be set for the models we used.

Reply:
OK. Then let's add an assert for irVersion_ >= 3, and another one for opsetVersion_ is found.